doc/manpages/bob-project.rst

```diff
@@ -167,6 +167,33 @@ to be passed on the command line *after* the package name.
     Add line to .config file. Can be used to specify preprocessor defines used by the QTCreator.
 
 
+Visual Studio Code project generator
+-----------------------------
+
+::
+
+    bob project vscode <package> [-h] [-u] 
+                            [--destination DEST] [--name NAME]
+                            [--exclude EXCLUDES] [-I ADDITIONAL_INCLUDES]
+
+The Visual Studio Code generator will generate a single .code-workspace file which could be opened in the Visual Studio Code. 
+
+The Visual Studio Code generator has the following specific options. They have to be
+passed on the command line *after* the package name.
+
+``--destination DEST``
+    Destination of project files.
+
+``--exclude EXCLUDES``
+    Packages will be marked as 'exclude from build' in eclipse. Usefull if indexer runs OOM.
```

Comment:
Cut'n'paste left over...

Reply:
Which generic options are really available for vscode? The only self defined options is "--sort".